lib/galaxy/datatypes/registry.py

```diff
@@ -141,7 +141,7 @@ def __import_module(full_path, datatype_module, datatype_class_name):
                 auto_compressed_types = galaxy.util.listify(elem.get('auto_compressed_types', ''))
                 sniff_compressed_types = galaxy.util.string_as_bool_or_none(elem.get("sniff_compressed_types", "None"))
                 if sniff_compressed_types is None:
-                    sniff_compressed_types = getattr(self.config, "sniff_compressed_dynamic_datatypes_default", True)
+                    sniff_compressed_types = getattr(self.config, "sniff_compressed_dynamic_datatypes", True)
```

Comment:
This can be configured/overridden on a per-datatype basis in the datatypes_conf.xml - so I think the old config value name is more appropriate.

Reply:
I removed the renaming and added the note about configuration to the description. Thanks for pointing this out.